composer.json

```diff
@@ -15,8 +15,8 @@
         "php": ">=5.6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6",
-        "orchestra/testbench": "^3.5"
+        "phpunit/phpunit": "^6.0|^7.0|^8.0",
```

Comment:
@franzose Here I had to change the requirements in the `composer.json` file to allow Composer to install the latest Laravel framework in accordance to the correct testbench version for the specific Laravel version. Without doing so it was keeping testing on Laravel 5.5 only

Reply:
It's okay for tests.

Reply:
I checked the report: https://travis-ci.org/github/franzose/ClosureTable/builds/675810551. I think we can test Laravel 5.5-5.8 on PHP 7.0-7.4. Now it's only PHP 7.2. What do you think?